app/controllers/users_controller.rb

```diff
@@ -21,6 +21,11 @@ def create
 
   def show
     @user = current_user
+    if current_user.admin?
+      redirect_to admin_dashboard_path
+    else
```

Comment:
we will want to refactor this so that before anything, we check if they are an admin, and then send them to their correct controller based on if they are an admin or a user; we don't want admins coming here and then being redirected to the admin show page, just send them directly to their show page from the admin users controller